### PR TITLE
Add faction colors for mini-factions.

### PR DESCRIPTION
**Pull request by plural**
Address #547 in this repo, at least.  NRDB ... doesn't appear to use these colors, but i know the discord bot does and we may use them in other places.

### CODE REVIEW

File: v2/factions.json [hunk 1]
```diff
@@ -78,7 +78,7 @@
   },
   {
     "code": "sunny-lebeau",
-    "color": "000000",
+    "color": "6E6E6E",
```
Comment (distributive) [inline v2/factions.json:81]:
```suggestion
    "color": "786196",
```

**Review by distributive — CHANGES_REQUESTED** [review]
They do actually appear on the site as profile colors:
image

Looking at this I think Sunny might be too close to the neutral color. I took a leaf from GLC's book and used the purple tone from her background:
image

**Comment by plural** [comment]
my local instance is not showing anything different here.  :/

Additionally, the css has its own, literal, copy of the colors.  :/

**Comment by distributive** [comment]
Classic NetrunnerDB

**Review by distributive — APPROVED** [review]

**Pull request merged**